engine/render/src/render/font_renderer.h

```diff
@@ -252,8 +252,9 @@ namespace dmRender
      * @param width max width. used only when line_break is true
      * @param line_break line break characters
      * @param metrics Metrics, out-value
+     * @return line count of text
```

Comment:
line_count is calculated one way or another and can be a valuable piece of info

Reply:
I think it should probably be a part of the text metrics struct since it's information about the text data :+1

Reply:
Cool. I'll just put it there then.